src/components/ArticleList/ArticleList.module.scss

```diff
@@ -22,3 +22,29 @@
 p.NoArticlesText {
   color: var(--text);
 }
+
```

Comment:
Appreciate your comment explaining why this lives here for now + that we don't use pagination anywhere else at the moment. Do you think it's worth adding a comment/TODO here for future us?